src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Write.ByteArray.cs

```diff
@@ -6,13 +6,24 @@ namespace System.Text.Json.Serialization
 {
     public static partial class JsonSerializer
     {
-        // Name \ feature is pending closure on API review
+        /// <summary>
+        /// Convert the value and return as a <see cref="System.Byte"/> array.
```

Comment:
What's `value` here?

Reply:
Changed to `A UTF-8 representation of the value.`